### PR TITLE
Perform create/delete actions in heartbeat view

### PR DESCRIPTION
**Pull request by leplatrem**
ref mozilla-services/readinglist#229

@Natim @ametaireau @tarekziade r?

(*we do more with less code* \o/)

### CODE REVIEW

File: cliquet/cache/__init__.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+import random
+
+
+_HEARTBEAT_DELETE_RATE = 0.5
+_HEARTBEAT_KEY = '__heartbeat__'
+_HEARTBEAT_TTL = 3600
```
Comment (Natim) [inline cliquet/cache/__init__.py]: ``_HEARTBEAT_TTL_SECONDS``

File: cliquet/cache/__init__.py [hunk 2]
```diff
@@ -21,7 +29,14 @@ def ping(self):
         :returns: `True` is everything is ok, `False` otherwise.
         :rtype: boolean
         """
-        raise NotImplementedError
+        try:
+            if random.random() > _HEARTBEAT_DELETE_RATE:
```
Comment (Natim) [inline cliquet/cache/__init__.py]: ``random.random() < _HEARTBEAT_DELETE_RATE``

In case of 0.5 it is the same but not with 0.3

**Comment by Natim** [comment]
LGTM

**Comment by ametaireau** [comment]
I like the idea of letting backends decide what's the best way to "ping" them, and this let's it open as a possibility, via inheritance.

+1 from me.

**Pull request merged**